diff --git a/Package.swift b/Package.swift
index 59c17ab..7bba7b1 100644
--- a/Package.swift
+++ b/Package.swift
@@ -86,9 +86,7 @@ let package = Package(
     products: [
         .library(name: "Crypto", targets: ["Crypto"]),
         .library(name: "_CryptoExtras", targets: ["_CryptoExtras"]),
-        /* This target is used only for symbol mangling. It's added and removed automatically because it emits build warnings. MANGLE_START
-            .library(name: "CCryptoBoringSSL", type: .static, targets: ["CCryptoBoringSSL"]),
-            MANGLE_END */
+        .library(name: "CCryptoBoringSSL", type: .static, targets: ["CCryptoBoringSSL"]),
     ],
     dependencies: [
         // Dependencies are added below so that they can be switched between local and absolute URLs
