diff --git a/Package.swift b/Package.swift
index 983e0d1..815e303 100644
--- a/Package.swift
+++ b/Package.swift
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.7
+// swift-tools-version: 5.10
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
@@ -16,16 +16,17 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-nio-ssh", from: "0.6.0"),
-        .package(url: "https://github.com/apple/swift-crypto.git", "1.0.0" ..< "2.1.0"),
-        .package(url: "https://github.com/apple/swift-nio-transport-services.git", "1.0.0" ..< "2.1.0"),
+        .package(url: "https://github.com/apple/swift-nio-ssh", from: "0.12.0"),
+        .package(path: "../swift-crypto"),
+        .package(url: "https://github.com/apple/swift-nio-transport-services.git", from: "1.26.0"),
     ],
     targets: [
         .target(
             name: "SSHClient",
             dependencies: [
                 .product(name: "NIOSSH", package: "swift-nio-ssh"),
-                .product(name: "_CryptoExtras", package: "swift-crypto"),
+                .product(name: "Crypto", package: "swift-crypto"),
+                .product(name: "CCryptoBoringSSL", package: "swift-crypto"),
                 .product(name: "NIOTransportServices", package: "swift-nio-transport-services"),
             ]
         ),
@@ -33,5 +34,6 @@ let package = Package(
             name: "SSHClientTests",
             dependencies: ["SSHClient"]
         ),
-    ]
+    ],
+    swiftLanguageVersions: [.v5]
 )
